client/src/modules/reducer/organization.js

```diff
@@ -35,7 +35,7 @@ export default createReducer(initialState, handlers)
 
 function requestOrganization(state, data) {
   const { payload: { organization } } = data
-  const { media, users } = organization
+  const { media } = organization
```

Comment:
This was removed in an earlier merge. There was a mediaReducer function for the list of users in the organization. Now there are no images in the user table. See https://github.com/iandusenbury/tableland/pull/65/files and add it back.

Reply:
Re-added the function and pushed up the fix.